bears/csv/CSVLintBear.py

```diff
@@ -17,6 +39,7 @@ class CSVLintBear:
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/dlzd1y6vnwx7kk1a17p2gk7ll'
```

Comment:
if CSVLint doesn't support some simpler format, you have to provide your own `process_output` and take over the regex parsing yourself. See http://api.coala.io/en/latest/Developers/Linter_Bears_Advanced.html#custom-processing-functions-with-process-output